### PR TITLE
Update MacOS installation information

### PR DESCRIPTION
**Pull request by pcolladosoto**
**Detailed description**
We have included information on how to install Rizin both through Homebrew and MacPorts on top of using the package file provided on the releases.

**Closing issues**
This PR is intended to add information on the installation procedures offered after closing issue #160.

### CODE REVIEW

**Review by XVilka — APPROVED** [review]

File: INSTALL.md [hunk 1]
```diff
@@ -30,15 +30,26 @@ anywhere because Rizin is compiled in a "portable" way, allowing moving the
 whole directory anywhere.
 
 ## MacOS
-On MacOS systems, Rizin can be installed through .pkg files attached to the
+On MacOS systems you can install Rizin in one of three ways.
+
+### Homebrew
+Rizin offers a Homebrew [formula](https://formulae.brew.sh/formula/rizin)
```
Comment (ret2libc) [inline INSTALL.md:45]: both here and in the next section i feel we should be more explicit on the fact that it is not us who provide these things, but they are maintained by the respective communities (homebrew and macports). We might help them keep them updated or fix things here and there, but we are not actively keeping them updated.

**Review by ret2libc — COMMENTED** [review]
Thanks a lot!

File: INSTALL.md (hunk 1 above)
Comment (pcolladosoto) [reply INSTALL.md:45]: I added some information tackling this topic. What do you think?

Comment (wargio) [reply INSTALL.md:45]: LGTM

**Review by wargio — APPROVED** [review]

**Review by ret2libc — APPROVED** [review]

**Comment by ret2libc** [comment]
Thanks! ;)

**Pull request merged**